examples/wifi-echo/server/esp32/main/gen/af-gen-event.h

```diff
@@ -49,8 +48,21 @@
     extern void emberAfPluginIasZoneServerManageQueueEventHandler(void);                                                           \
     extern void emberAfPluginNetworkSteeringFinishSteeringEventHandler(void);                                                      \
     extern void emberAfPluginReportingTickEventHandler(void);                                                                      \
-    extern void emberAfPluginScanDispatchScanEventHandler(void);                                                                   \
     extern void emberAfPluginUpdateTcLinkKeyBeginTcLinkKeyUpdateEventHandler(void);                                                \
+    static void networkEventWrapper(EmberEventControl * control, EmberAfNetworkEventHandler handler, uint8_t networkIndex)         \
```

Comment:
Are these changes due to changes in the silabs Simplicity Studio or SDK versions we're using?  They don't seem relevant to the actual changes being made here....

Reply:
I believe they are. I'm using a different version of the gecko sdk than the one that is used to generate the initial version. For the record I do used 3.0.0, and the initial version was 2.7.0 . 

I looked into the UI to see if there is something activated that I missed but couldn't find anything relevant.